frontend/src/views/admin/SubscriberPanelView.vue

```diff
@@ -67,6 +71,13 @@ const filteredSubscribers = computed(() => subscribers.value.map((subscriber) =>
     value: subscriber.time_deleted ? 'Enable' : 'Disable',
     disabled: !subscriber.time_deleted && subscriber.email === user.data.email,
   },
+  hardDelete: {
+    type: TableDataType.Button,
+    buttonType: TableDataButtonType.Caution,
+    value: 'Hard Delete',
+    disabled: !subscriber.time_deleted || subscriber.email === user.data.email,
+    tooltip: 'Completely remove the user from the database. This is the same as deleting someones account!! You must disable the account first.',
```

Comment:
Oh also added tooltips as a possibility for buttons

Reply:
Nice. I suggest we localize those too.